src/modules/currency/currencies.ts

```diff
@@ -0,0 +1,135 @@
+import BigNumber from 'bignumber.js';
+import memoize from 'memoize-one';
+
+interface ResolvedNumberFormatOptions {
+  numberingSystem: string;
+  style: string;
+  currency?: string;
+  currencyDisplay?: string;
+  minimumIntegerDigits: number;
+  minimumFractionDigits: number;
+  maximumFractionDigits: number;
+  minimumSignificantDigits?: number;
+  maximumSignificantDigits?: number;
+  useGrouping: boolean;
```

Comment:
I think we shouldn't mention values that we're not relying upon. Or we should just reuse `NumberFormatOptions` type